request-fetch-adapter.js

```diff
@@ -4,10 +4,11 @@ module.exports = {
       callback = requestBody;
       requestBody = null;
     }
-
-    var headers = {
+    
+    const headers = {
       "Accept": "application/json",
-      "Content-Type": "application/json"
+      "Content-Type": "application/json",
+      "X-SDK-Version": getSDKVersion()
```

Comment:
Should we include "CodePush" in this name somewhere? (e.g. "X-CodePush-SDK-Version")